firmware/console/status_loop.cpp

```diff
@@ -696,7 +696,7 @@ void updateTunerStudioState(TunerStudioOutputChannels *tsOutputChannels DECLARE_
 
 	if (hasMapSensor(PASS_ENGINE_PARAMETER_SIGNATURE)) {
 		float mapValue = getMap();
-		tsOutputChannels->veValue = veMap.getValue(rpm, mapValue);
+		tsOutputChannels->veValue = engine->engineState.currentVE * 100.0f;//veMap.getValue(rpm, mapValue);
```

Comment:
We probably do not need dead "//veMap.getValue(rpm, mapValue);" here

also kind of in the direction of https://refactoring.com/catalog/replaceMagicNumberWithSymbolicConstant.html - can we maybe help a helper macro with a clear name? I assume we convett from 0..1 to human-readable 0..100% here?

maybe this should be done not in the firmware but in TS file? I am not sure if there is a solution for logging.